boards/arm/frdm_k64f/frdm_k64f-pinctrl.dtsi

```diff
@@ -0,0 +1,130 @@
+/*
+ * NOTE: Autogenerated file by kinetis_signal2dts.py
+ *       for MK64FN1M0VLL12/signal_configuration.xml
+ *
+ * Copyright (c) 2022, NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <nxp/kinetis/MK64FN1M0VLL12-pinctrl.dtsi>
+
+&pinctrl {
+	i2c0_default: i2c0_default {
+		group0 {
+			pins = <&I2C0_SCL_PTE24
+				&I2C0_SDA_PTE25>;
```

Comment:
note: I think `lower_case` nodes would align better with DT style. `UPPER_CASE` is usually for macros/consts. @mbolivar-nordic can you comment on this one please?

Reply:
I agree here that lower case nodes would be best, the reason for these node names is that NXP's HAL already has defined pinmux options for Kinetis parts using this style, so those files would have to be regenerated with lowercase node naming. (See the K64 pinmux defintion [here](https://github.com/zephyrproject-rtos/hal_nxp/blob/master/dts/nxp/kinetis/MK64FN1M0VLL12-pinctrl.dtsi))

Reply:
Right, galak did it this way if I recall correctly, so the ship has sailed.

Reply:
@danieldegrasse it looks like you can just use definitions instead of nodes for this case (seems like 32-bit should be enough to store stuff like `nxp,kinetis-port-pins = < 13 2 >;`.

Reply:
You definitely could- the two values combined will fit into 32 bits easily. The existing pinctrl definitions in NXP's HAL would have to be regenerated, but there's no reason that can't be done. @mbolivar-nordic, do you think regenerating the pinctrl definitions avoid using nodes here would be best? The final implementation would likely end up looking similar to the one introduced in #39974. The main benefits would be slightly smaller code size (since the `pinctrl_soc_t` typedef would be a `uint32_t`), and better alignment with the pinctrl implementation guide.

Reply:
I think that using definitions would be better here, nodes are not really required in this case. AFAIK, the generated nodes have never been used in reality, so it's a good time to make this kind of decision.